moodlecli/moodle.py

```diff
@@ -104,13 +104,16 @@ def _get(self, service_function, data=None):
         return check_for_moodle_error(res)
 
     def copy_course(
-        self, source_id, course_name, course_shortname, course_category_id
+        self, source_id, course_name, course_shortname, course_category_id,
+        include_users
```

Comment:
Minor thought: If you wanted, you could make this have a default value of `False` so the method could be invoked without passing the argument unless necessary.

Reply:
Ah, that would have been easier than changing all the invocations yup, I think I'll do that.